website/docs/developers/android/build.md

```diff
@@ -0,0 +1,55 @@
+---
+id: build-app
+title: Creating a build/development environment for the Android App
+sidebar_label: Building Android App
+---
+
+
+## Build instructions
+
+If you would like to develop this application we'd love your help!  These build instructions are brief 
+and should be improved, please send a PR if you can.
+
+* Use Android Studio 4.1.2 to build/debug (other versions might work but no promises)
+* Use "git submodule update --init --recursive" to pull in the various submodules we depend on
+* There are a few config files which you'll need to copy from templates included in the project.
+Run the following commands to do so:
+
+```
+        rm ./app/google-services.json
+        cp ./app/google-services-example.json ./app/google-services.json
+        rm ./app/src/main/res/values/mapbox-token.xml
+        cp ./app/special/mapbox-token.xml ./app/src/main/res/values/
+        rm ./app/src/main/res/values/curfirmwareversion.xml
+        cp ./app/special/curfirmwareversion.xml ./app/src/main/res/values/
+```
+
+* (unfortunately) you need to get a (free) mapbox developer token [here](https://docs.mapbox.com/android/maps/guides/install/) and put that token in your user gradle.properties.
+```
+~/development/meshtastic/MeshUtil$ cat ~/.gradle/gradle.properties
+MAPBOX_DOWNLOADS_TOKEN=sk.yourtokenherexxx
+```
+
+* Now you should be able to select "Run / Run" in the IDE and it will happily start running on your phone
+or the emulator.  Note: The emulators don't support bluetooth, so some features can not be used in
+that environment.
```

Comment:
should we break these line breaks?

Reply:
actually, the note could probably be added to an admonition

Reply:
Good point, done